src/FcmChannel.php

```diff
@@ -66,15 +68,20 @@ public function send($notifiable, Notification $notification)
         $responses = [];
 
         try {
-            if (count($tokens) === 1) {
-                $responses[] = $this->sendToFcm($fcmMessage, $tokens[0]);
-            } else {
-                $partialTokens = array_chunk($tokens, self::MAX_TOKEN_PER_REQUEST, false);
+            $partialTokens = array_chunk($tokens, self::MAX_TOKEN_PER_REQUEST, false);
```

Comment:
I presume then we can remove the `sendToFcm` method if it's no longer used?

Reply:
I wasn't sure if someone might created a subclass of `FcmChannel` accessing the `sendToFcm` method. 
But if we really introduce breaking changes we could definitly remove that method, since we don't use it anymore from within the package.

Reply:
Yeah, let's ditch it. If people require it they can stick to an older version.